test/utils/test.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
+MIIFjjCCA3YCCQCVV/TwiFnSozANBgkqhkiG9w0BAQsFADCBiDELMAkGA1UEBhMC
```

Comment:
Is there a better way to do this? Maybe instead I can add this to a string and write this to a file in the test?

That way we don't bloat code-server with new files